File: server/test_lutris.js
async function testLutris(q) {
  try {
    const response = await fetch(`https://lutris.net/api/games?search=${encodeURIComponent(q)}`);
    const data = await response.json();
    console.log(`--- Results for "${q}" (Total: ${data?.results?.length || 0}) ---`);
    if (data && data.results) {
      data.results.slice(0, 5).forEach(item => {
        console.log({
          name: item.name,
          slug: item.slug,
          coverart: item.coverart,
          banner_url: item.banner_url
        });
      });
    }
  } catch (err) {
    console.error('Error:', err.message);
  }
}

async function run() {
  await testLutris("Grand Theft Auto V");
  await testLutris("Fortnite");
}

run();

File: test_lutris.py
import requests

try:
    response = requests.get("https://lutris.net/api/games?search=GTA", headers={
        "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
    })
    print("Status Code:", response.status_code)
    data = response.json()
    print("Keys:", data.keys() if isinstance(data, dict) else "Not a dict")
    if isinstance(data, dict) and "results" in data:
        print("Results length:", len(data["results"]))
        if len(data["results"]) > 0:
            print("First item keys:", data["results"][0].keys())
            print("First item preview:", data["results"][0])
except Exception as e:
    print("Error:", e)
